python/hail/tests/test_methods.py

```diff
@@ -370,6 +370,8 @@ def test_split_multi_hts(self):
         with self.assertRaises(utils.FatalError):
             hl.split_multi_hts(ds)
 
+        ds1._force_count_rows()
```

Comment:
Don't see what this is doing.  In particular, _same is called on ds1 above which will force it to be evaluated.